aces/Loris
# [Survey Accounts] Batch send surveys

## Brief summary of changes

This PR changes the instrument select in the "add survey" section to a multi-select


#### Testing instructions (if applicable)

1. Go to Survey Accounts
2. Click on Add survey
3. The instrument selection should be a multi-select

#### Link(s) to related issue(s)

* Resolves #8984
